### DISCUSSION TITLE
[QUESTION] How to load HTML string in the Canvas in the React App?

### DISCUSSION THREAD

**Discussion by harryandriyan** [GrapesJS/grapesjs#4867] [Q&A]
Hi, all I'm using https://github.com/thanhtunguet/grapesjs-react wrapper in my app.

I've tried to load HTML string from the remote URL to the canvas and found that 
`editor.setComponent(htmlString)` will work, but in the `onInit` props function the `editor` param doesn't has `setComponent` function.

Thanks!

**Comment by artf** [comment]
There is no `setComponent` method on the editor instance but [`setComponents`](https://grapesjs.com/docs/api/editor.html#setcomponents)
